lib/jekyll-import/importers/mt.rb

```diff
@@ -58,15 +65,24 @@ def self.specify_options(c)
       def self.process(options)
         options  = default_options.merge(options)
 
+        engine   = options.fetch('engine')
         dbname   = options.fetch('dbname')
         user     = options.fetch('user')
         pass     = options.fetch('password', "")
         host     = options.fetch('host', "localhost")
+        port     = options.fetch('port')
```

Comment:
That would be a side effect of changing this, but I believe you will get a KeyError if you do not specify port in the command line when you run this because the port key will not exist. See the description here:

http://ruby-doc.org/core-2.2.0/Hash.html#method-i-fetch